parse/lex.go

```diff
@@ -629,8 +629,8 @@ func lexReservedBody(l *lexer) stateFn {
 		case v == '\\': // Reserved escape
 			v = l.next()
```

Comment:
Attn: the `v` gets reassigned. Won't it break the whole `switch`? Earlier (line 250) in similar case you're defining a new variable.

Reply:
Good spot. Renamed to `next`.